src/guc.c

```diff
@@ -55,6 +55,13 @@ static const struct config_enum_entry remote_data_fetchers[] = {
 	{ NULL, 0, false }
 };
 
+static const struct config_enum_entry hypertable_distributed_types[] = {
+	{ "auto", HYPERTABLE_DIST_AUTO, false },
+	{ "local", HYPERTABLE_DIST_LOCAL, false },
+	{ "distributed", HYPERTABLE_DIST_DISTRIBUTED, false },
```

Comment:
I am wondering if these should be "true" and "false" since this GUC is the default for the "distributed" boolean parameter. I think "true" and "false" also makes more sense given the GUC name, e.g., `hypertable_distributed_default=true;` rather than setting `hypertable_distributed_default=local` seems a bit counter intuitive. 


```suggestion
	{ "false", HYPERTABLE_DIST_LOCAL, false },
	{ "true", HYPERTABLE_DIST_DISTRIBUTED, false },
```

Reply:
This can't be boolean and having booleanish values as strings is confusing when there is an actual boolean type exists.

Boolean value is either true of false (there are no notion of undefined in gucs), but having false by default will break compatability and will force `create_hypertable(replication_factor)` to emit error.